test/tobs/test_list.py

```diff
@@ -173,23 +173,23 @@ def test_list_cached_references(capsys):
     ListScript("crds.list --cached-references --full-path")()
```

Comment:
Might need to split this into two separate tests and pass in the appropriate fixture i.e. 'test_list_cached_references_jwst(jwst_shared_cache_state, capsys)` 

'test_list_cached_references_hst(default_shared_state, capsys)` 

NB default_shared_state defaults to hst-related settings